Resources/Prototypes/Body/Parts/skeleton.yml

```diff
@@ -0,0 +1,224 @@
+# TODO BODY: Part damage
+- type: entity
+  id: PartSkeleton
+  parent: BaseItem
+  name: "skeleton body part"
+  abstract: true
+  components:
+  - type: Damageable
+    damageContainer: Biological
+
+- type: entity
+  id: TorsoSkeleton
+  name: "skeleton torso"
+  parent: PartSkeleton
+  components:
+    - type: Sprite
+      netsync: false
+      sprite: Mobs/Species/Skeleton/parts.rsi
+      state: "torso_m"
+    - type: Icon
+      sprite: Mobs/Species/Skeleton/parts.rsi
+      state: "torso_m"
+    - type: BodyPart
+      partType: Torso
+      size: 14
+      compatibility: Skeleton
+#    criticalThreshold: 100
+#    deadThreshold: 150
+
+- type: entity
+  id: HeadSkeleton
+  name: "skull"
+  description: Alas poor Yorick...
+  parent: PartSkeleton
+  components:
+    - type: Sprite
+      netsync: false
+      sprite: Mobs/Species/Skeleton/parts.rsi
+      state: "skull_icon"
+    - type: Icon
+      sprite: Mobs/Species/Skeleton/parts.rsi
+      state: "skull_icon"
+    - type: BodyPart
+      partType: Head
+      size: 7
+      compatibility: Skeleton 
+    #Unique stuff the skull has for one a skelly gets "boned" nyeheheh
+    - type: SkeletonBodyManager
```

Comment:
the name for this component is kinda weird. again like i said it should be merged with the `TransferOnGib` and probably given a better name

Reply:
also there's already an existing system where control gets sent to the brain when an entity is gibbed right? can that be used at all for this stuff instead. idr how it works tbh

Reply:
i looked into brain component and it seems to be kinda wacky. Not only does it not work for bodyparts, only for organs, it also allows the player to freely move from ghost into the brain, which is obviously broken as hell for a living character.